apps/src/pythonlab/pyodideWorkerManager.ts

```diff
@@ -72,13 +75,15 @@ const setUpPyodideWorker = () => {
         getStore().dispatch(setAndSaveSource(message));
         break;
       case 'error':
+        getStore().dispatch(setHasError(true));
```

Comment:
Oh, nice! For some reason, without looking, I assumed setting this from inside the pyodideWorkerManager wouldn't work 😅